scripts/ipxe-dts.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+error_exit() {
+    _error_msg="$1"
+    echo "$_error_msg"
+    exit 1
+}
+
+error_check() {
+    _error_code=$?
+    _error_msg="$1"
+    [ "$_error_code" -ne 0 ] && error_exit "$_error_msg : ($_error_code)"
+}
+
```

Comment:
@WiktorG351 please define

SCRIPTDIR=$(readlink -f "$(dirname "$0")")

and then use this variable as a path reference used in other commands

Reply:
@TomaszAIR done [here](https://github.com/Dasharo/meta-dts/pull/96/commits/eb5dd73352f83f419cdf51aef35e169b5f3dc08c#diff-50ccba31fdf15d04ffc9cadbcbbfc4ceebd19e4076523602a1ee758e9a72cc68R3)